foreman/data_refinery_foreman/surveyor/management/commands/survey_sra.py

```diff
@@ -20,16 +24,29 @@ def add_arguments(self, parser):
         parser.add_argument(
             "--file",
             type=str,
-            help=("An optional file listing accession codes.")
+            help=("An optional file listing accession codes. s3:// URLs are also accepted.")
         )
 
     def handle(self, *args, **options):
         if options["accession"] is None and options["file"] is None:
             logger.error("You must specify accession or input file.")
             return 1
         if options["file"]:
-            with open(options["file"]) as file:
-                for acession in file:
```

Comment:
This line has a typo: `acession` should be `accession`. It has been corrected in this PR too.